.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
```

Comment:
`bug`, maybe

Reply:
thank you, added and also `enhancement` to feature request